lib/tool_belt/changelog.rb

```diff
@@ -35,9 +36,9 @@ def generate_entry(issue)
 
       list_item = title_string
       list_item += issue_string
-      list_item += ", "
 
-      list_item += hash_string(issue)
+      list_item += ", "
+      list_item += hash_string(issue) unless @omit_gh_link
```

Comment:
No link also means no revision SHA? Seems like we would just want to omit the link part (the parenthesis).